docs/server-side/v2/api-reference/Rooms/create-via-api.mdx

```diff
@@ -95,3 +95,17 @@ To know more about recording please visit [Recording](/server-side/v2/Destinatio
 
 The access keys should have read(GetObject) and write(PutObject) permissions for the storage bucket.
 For more details check - https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html
+
+### Errors
+
```

Comment:
can we add an example error payload here to show the structure of the API response for error scenarios?